src/lib/components/blog/blog-entry.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { makeBlogPostPageUrl } from '$lib/public/blog/urls';
+  import { readingTime } from '$lib/public/blog/utils';
+  import { Temporal } from '@js-temporal/polyfill';
+  import Labels from '../../../routes/blog/labels.svelte';
+
+  export let post: any;
+  export let selectedLabels: any;
+  console.log(post);
+</script>
+
+<div class="post">
+  <a href={makeBlogPostPageUrl(post.id)}>
+    <div class="author">
+      <a class="author-avatar" href={post.author.url}>
+        <img src={post.author.picture} alt="" />
+      </a>
+      <div>
+        <a href={post.author.url}>{post.author.displayname}</a>
+      </div>
+    </div>
+    <a href={makeBlogPostPageUrl(post.id)}>
+      <h2 class="headers">{post.title}</h2>
+    </a>
+    <div class="markdown-body">
+      {@html post.html}
+    </div>
+    <p class="read-time">
+      {post.createdAt &&
+        Temporal.Instant.from(post.createdAt).toLocaleString('en-US', {
+          calendar: 'gregory',
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        })} •
+      {readingTime(post.html)} min read
+      <Labels data={post.labels} {selectedLabels} />
+    </p>
+  </a>
+</div>
+
+<style>
+  .post {
+    padding: 2em;
+  }
+  a {
```

Comment:
Nit: Space out CSS blocks. 
Note: Adding empty lines in this repository should be covered in our style guide.

Reply:
addressed